.github/workflows/update-agent-changelog.yml

```diff
@@ -0,0 +1,71 @@
+name: Update the Agent changelog
+
+on:
+  push:
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+'
```

Comment:
Interesting. So the expectation is that we'll open the PR once the tag is pushed and then merge/adjust it after the release has been deployed and published, right?

For context, us pushing the tag is only the first step in the "final release", this tag must also get deployed and then pulbished.

Reply:
I don't know what `this tag must also get deployed and then pulbished.` means 🤔 In integrations-core, we create the tag 7.50.1 once the agent is released. Then, we update the changelogs right after. Pushing the very first step of the process on our side, and I automated the rest to open the PR for us

Reply:
Ah, nvm. I checked with Kacper. Your understanding is correct. It's safe to open a PR right after we push the tag. We should still wait for green light from Kacper to actually merge this PR, but there's rarely too much delay with that.

Reply:
I'll write that down in our docs 🙂